actionview/test/template/form_options_helper_test.rb

```diff
@@ -851,6 +851,15 @@ def test_select_with_nil_and_selected_option_as_nil
     )
   end
 
+  def test_select_with_array
+    @continent = Continent.new
+    @continent.countries = ["Africa", "Europe"]
+    assert_dom_equal(
+      %(<select name="continent[countries]" id="continent_countries"><option selected="selected" value="Africa">Africa</option>\n<option selected="selected" value="Europe">Europe</option>\n<option value="America">America</option></select>),
+      select("continent", "countries", %W(Africa Europe America), { multiple: true })
```

Comment:
FYI this test is wrong (the `multiple` attribute isn't doing what it should), https://github.com/rails/rails/pull/46629 includes a fix.